packages/schemas/src/foundations/jsonb-types/file-upload.ts

```diff
@@ -0,0 +1,12 @@
+import { number, object, string, type z } from 'zod';
+
+export const uploadFileGuard = object({
+  filepath: string(),
+  mimetype: string(),
+  originalFilename: string(),
+  size: number(),
```

Comment:
Can you add some detailed explanations of how these fields will be used?

Reply:
agreed. you should define the type first, instead of using `z.infer`

Reply:
This is an existing uploadFileGuard, which was originally used for image uploading. I just moved it to the schemas, thought that can be re-used in custom UI assets upload.

Reply:
Script updated, no more "custom-ui-assets" table needed, therefore this jsonb type is also removed